### PR TITLE
Documentation update for Signal Messenger integration for adding URL attachment support

### PR DESCRIPTION
**Pull request by burnsie-la**
Added documentation for a new feature I am proposing to this integration - https://github.com/home-assistant/core/pull/62311

## Proposed change
I am proposing a feature to the Signal Integration to allow sending of attachments in Signal messages from a URL.  This is the documentation update for this feature.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/62311

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/signal_messenger.markdown [hunk 1]
```diff
@@ -90,3 +90,16 @@ action:
...
+### Text message with an attachment from a URL
+
+```yaml
+...
+action:
+  service: notify.NOTIFIER_NAME
+  data:
+    message: "Person detected on Front Camera!"
+    data:
+      urls:
```
Comment (ianByrne) [inline source/_integrations/signal_messenger.markdown:108]:
```suggestion
            verifyUrls: false
            urls:
```

File: source/_integrations/signal_messenger.markdown [hunk 2]
```diff
@@ -90,3 +90,16 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
```
Comment (ianByrne) [inline source/_integrations/signal_messenger.markdown:95]:
```suggestion

Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifyUrls` to `False` to ignore SSL errors (default `True`)

```

**Review by ianByrne — CHANGES_REQUESTED** [review]

File: source/_integrations/signal_messenger.markdown [hunk 3]
```diff
@@ -90,3 +90,19 @@ action:
...
+```yaml
+...
+action:
+  service: notify.NOTIFIER_NAME
+  data:
+    message: "Person detected on Front Camera!"
+    data:
+            verifyUrls: false
+            urls:
```
Comment (ianByrne) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
      verifySsl: False
      urls:
```

File: source/_integrations/signal_messenger.markdown [hunk 4]
```diff
@@ -90,3 +90,19 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifyUrls` to `False` to ignore SSL errors (default `True`)
```
Comment (ianByrne) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)
```

**Review by ianByrne — CHANGES_REQUESTED** [review]

File: source/_integrations/signal_messenger.markdown [hunk 5]
```diff
@@ -90,3 +90,19 @@ action:
...
+Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)
+
+```yaml
+...
+action:
+  service: notify.NOTIFIER_NAME
+  data:
+    message: "Person detected on Front Camera!"
+    data:
+      verifySsl: False
```
Comment (frenck) [inline source/_integrations/signal_messenger.markdown]: `verifySsl` isn't following our standards; it should be:

```suggestion
      verify_ssl: false
```

Additionally, I'm confused why Home Assistant needs to verify SSL, to begin with?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/signal_messenger.markdown [hunk 6]
```diff
@@ -90,3 +90,19 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)
```
Comment (ianByrne) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_urls) list.

Note there is a 50mb size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `False` to ignore SSL errors (default `True`)
```

**Review by ianByrne — CHANGES_REQUESTED** [review]

File: source/_integrations/signal_messenger.markdown [hunk 7]
```diff
@@ -90,3 +90,21 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_urls) list.
```
Comment (frenck) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](/docs/configuration/basic/#allowlist_external_urls) list.
```

File: source/_integrations/signal_messenger.markdown [hunk 8]
```diff
@@ -90,3 +90,21 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](/docs/configuration/basic/#allowlist_external_urls) list.
+
+Note there is a 50mb size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `False` to ignore SSL errors (default `True`)
```
Comment (frenck) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
Note there is a 50MB size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `false` to ignore SSL errors (default `true`).
```

File: source/_integrations/signal_messenger.markdown [hunk 9]
```diff
@@ -90,3 +90,21 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](/docs/configuration/basic/#allowlist_external_urls) list.
```
Comment (frenck) [inline source/_integrations/signal_messenger.markdown]:
```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [`allowlist_external_urls`](/docs/configuration/basic/#allowlist_external_urls) list.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**